src/GitHubParserHook.php

```diff
@@ -43,14 +53,58 @@ private function setFields( ProcessingResult $result ) {
 		$this->fileName = $params['file']->getValue();
 		$this->repoName = $params['repo']->getValue();
 		$this->branchName = $params['branch']->getValue();
+
+		foreach ( self::$syntaxHighlightFields as $val ) {
+			if ( isset( $params[$val] ) ) {
+				$this->$val = $this->cleanField( $params[$val]->getValue() );
```

Comment:
Time travel bunny pretends to not know what you are talking about

![timebunny](https://cloud.githubusercontent.com/assets/146040/24390103/da1c3e3e-1386-11e7-86f2-e9dc5801288f.jpg)